lib/blueberry_rails/app_builder.rb

```diff
@@ -175,17 +174,15 @@ def configure_simple_form
 
     def replace_users_factory
       remove_file 'spec/factories/users.rb'
-      template 'users_factory.rb', 'spec/factories/users.rb'
+      copy_file 'users_factory.rb', 'spec/factories/users.rb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.